test/validation/api/crypto/odp_crypto_test_inp.c

```diff
@@ -527,6 +527,17 @@ static void alg_test_execute(const alg_test_param_t *param)
 		     param->op == ODP_CRYPTO_OP_ENCODE))
 			continue;
 
+		/*
+		 * Checking against wrong digest is meaningless for AES-CBC +
+		 * HMAC-SHA chained operation. As per Armv8 crypto library
+		 * specification, a new digest is generated for decryption too.
+		 */
```

Comment:
> "Checking against wrong digest is meaningless for AES-CBC + HMAC-SHA
> chained operation."

Why would it be meaningless? There would be no point in doing SHA if the HMAC was not checked. And the validation test should check that an ODP implementation notices an incorrect MAC.

> As per Armv8 crypto library specification, a new digest is generated
> for decryption too.

I could not find such specification (can you provide a pointer?), but that is irrelevant since this code is ODP API validation test that does not have anything directly to do with the ARM crypto library. An ODP implementation must adhere to the ODP API spec and hide possible peculiarities of the underlying crypto library used in the implementation.